src/components/NewBookForm.js

```diff
@@ -34,15 +34,15 @@ const NewBookForm = () => {
           className="input titleInput"
           placeholder="Book title"
           value={title}
-          onChange={(e) => setTitle(e.target.value)}
+          onChange={(event) => setTitle(event.target.value)}
         />
         <input
           type="text"
           name="author"
           className="input authourInput"
           placeholder="Author"
           value={author}
-          onChange={(e) => setAuthor(e.target.value)}
+          onChange={(event) => setAuthor(event.target.value)}
         />
         <button className="AddBook" type="submit">ADD BOOK</button>
```

Comment:
- [ ] As part of the project requirements please make sure to create a custom `<Button/>` component instead of using the default `<button>` tag here and pass the event handler for adding a book as a prop. I noticed you're using the form `onSubmit` attribute but you should remove it so the button handles the submission.